### ISSUE TITLE
Zip code is a poor indicator of location to New Arrivals... should be city

### ISSUE THREAD

**Issue by FivelFred** [justarrived/just-match-frontend#546]
Would be better to have "area" or city as indicator. 

![skarmavbild 2016-06-10 kl 14 26 45](https://cloud.githubusercontent.com/assets/16607246/15964166/64b9ba12-2f17-11e6-9e96-5d824f2afc9b.png)

**Comment by buren** [comment]
@FivelFred out of curiosity why is zip a bad indicator?

**Comment by FivelFred** [comment]
Because few people can relate to zip code, especially people coming from other countries. 
Skickat från min Samsung-enhet

-------- Originalmeddelande --------
Från: Jacob Burenstam <notifications@github.com> 
Datum: 06-11-2016  18:28  (GMT+01:00) 
Till: justarrived/just-match-frontend <just-match-frontend@noreply.github.com> 
Kopia: FivelFred <fredricghatan@gmail.com>, Mention <mention@noreply.github.com> 
Rubrik: Re: [justarrived/just-match-frontend] Zip code is a poor indicator of
  location to New Arrivals... should be city (#546) 

@FivelFred out of curiosity why is zip a bad indicator?

**Comment by buren** [comment]
@FivelFred yeah thats a good point. The challenge though is that city is __very__ imprecise, Stockholm doesn't say much in terms of location. It could also cause ambiguity, for example there are numerous "Strandvägen" in Stockholm and without the zip we have no idea which one of them is the one the users means.. 

 I'm not sure what the best thing would be here. Either accept the ambiguity or maybe add some help text or link to an explanation? 

@camillaryd, @pstenstrm any ideas of how we could improve this?
